src/modules/keyboardmanager/test/MockedInput.cpp

```diff
@@ -70,6 +70,7 @@ UINT MockedInput::SendVirtualInput(UINT cInputs, LPINPUT pInputs, int cbSize)
                     keyboardState[VK_RCONTROL] = false;
                     break;
```

Comment:
I think this was actually a mistake and the break should be after the if blocks, so an explicit fallthrough is not required. It didn't affect the tests since it was doing a redundant assignment which was done in an earlier statement, but a fallthrough was not the expected behavior. Good catch!

Reply:
You want me to close my PR and you fix appropriately?